File: 프로그래머스/0/120815. 피자 나눠 먹기 （2）/피자 나눠 먹기 （2）.js
function solution(n) {
    //10명이면 30조각 -> 3조각 씩 
    //4명이면 12조각 -> 3조각 씩
    let pizza = 1; //피자 판 수
    
    while((pizza * 6 ) % n !== 0){
        pizza+=1;
    }
    return pizza;
}

File: 프로그래머스/0/120815. 피자 나눠 먹기 （2）/피자 나눠 먹기 （2）.java
class Solution {
    public int solution(int n) {
        // n과 6의 최소공배수를 구하고, 그걸 6으로 나눠 피자 판 수 구하기
        int num = gcd(6,n);
        int answer = num * (n/num) * (6/num);
            return answer/6;
    }

    // 최대공약수(GCD) 함수
    private static int gcd(int a, int b) {
        while (b != 0) {
            int tmp = a % b;
            a = b;
            b = tmp;
        }
        return a; //이게 최대공약수가 된다.
    }

}
